### PR TITLE
Related

### PR DESCRIPTION
**Pull request by Danwhy**
Gets related items from database

Returns related items in api response and html response. Currently set to 24 items, but will be variable with a query in the future.

@SimonLab Can you pull this down and check the tests run fine for you, then have a look over it please?

### CODE REVIEW

File: lib/sort-related-items.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const config = require('../config');
```
Comment (SimonLab) [inline lib/sort-related-items.js]: I think we could do the logic of the function by using aggregations on the request.

**Comment by SimonLab** [comment]
We need to start adding more documentation to be able to follow the flow of the logic of the code. We also should add more jsdoc on the functions to help us to know quickly what is the role of these functions. Otherwise LGTM

File: lib/jsonapi-response.js [hunk 2]
```diff
@@ -60,58 +59,90 @@ function getRelationships (data) {
...
+  return {
+    type: el.type,
+    id: el.id
+  };
+}
+
+function getIncluded (data, relationships, config, relatedItems) {
   if (relationships) {
-    const included = [];
+    var included = [];
```
Comment (nelsonic) [inline lib/jsonapi-response.js]: there are a few changes from `const` to `var` but not _universally_ applied. is this _deliberate_ ?

Comment (Danwhy) [inline lib/jsonapi-response.js]: Based on your previous suggestion not to use const for arrays/objects that are being added to, I've changed any I've noticed from before that were const

File: test/client/end-to-end.clienttest.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ module.exports = {
       .assert.urlEquals('http://localhost:8000/search?q=ada')
       .click('.resultcard')
       .assert.urlEquals('http://localhost:8000/people/smgc-people-38764')
+      .waitForElementVisible('.record-top__title', 1000)
```
Comment (nelsonic) [inline test/client/end-to-end.clienttest.js]: @Danwhy you can set a _global_ value for Nightwatch to wait for e.g: https://github.com/dwyl/learn-nightwatch/blob/master/nightwatch.conf.BASIC.js#L23 that way you can leave the second argument out and nightwatch will wait the global amount as a maximum.

**Comment by nelsonic** [comment]
LGTM! 🎉

**Pull request merged**